src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -988,10 +982,7 @@ protected DynamoModel(IStartConfiguration config)
                             {
                                 if (loadedExtension is IExtension)
                                 {
-                                    if (loadedExtension is LinterExtensionBase loadedLinter)
-                                    {
-                                        loadedLinter.InitializeBase(this.LinterManager);
-                                    }
```

Comment:
I know this is not your code but it looks like it's duplicate code. Isn't this already happening in lines 970-980, or is there some difference between an unloaded and a loaded extension?